lib/messagebird.js

```diff
@@ -226,6 +226,69 @@ module.exports = function (accessKey, timeout) {
       }
     },
 
+    mms: {
+
+      /**
+       * Get a mms message
+       *
+       * @param {String} id
+       * @param {Function} callback
+       * @return {void}
+       */
+      read: function (id, callback) {
+        httpRequest('GET', '/mms/' + id, callback);
+      },
+
+      /**
+       * Send a mms message
+       *
+       * @param {Object} params
+       * @param {Function} callback
+       * @return {void}
+       */
+      create: function (params, callback) {
+        if (params.recipients instanceof Array) {
+          params.recipients = params.recipients.join(',');
+        }
+        httpRequest('POST', '/mms', params, callback);
+      },
+
+      /**
+       * Lists mms messages. Pagination is optional. If a limit is set, an
+       * offset is also required.
+       *
+       * @param {Number} limit
```

Comment:
If this values are optional it would be good to mark them as optional in the JSDOC. [docs](http://usejsdoc.org/tags-param.html)

```suggestion
       * @param {Number=} limit
```

But accutally, in JS there is the data type `Number` and `number`. The one with the uppercase first character is called "Object Type" (There is also `String` and `Boolean`) but the plain type should be used all the time. There is no* reason to use object types since it could include unexpected behavior. 

To make really sure it's meant as plain type, write it as lowercase type:

```suggestion
       * @param {number=} limit
```

Same is true for the other typings.

*) there is one. Just ping me for an explanation 😸